api/pom.xml

```diff
@@ -50,26 +63,22 @@
                 <configuration>
                     <groups>
                         <group>
-                            <title>JMS API Documentation</title>
+                            <title>Jakarta Messaging API Documentation</title>
                             <packages>
-                                javax.jms
+                                javax.messaging
                             </packages>
                         </group>
                     </groups>
                 
-                    <!-- Specifies the text to be placed at the bottom of each output file. -->
-                    <bottom>Copyright 1997-2018 Oracle America Inc. All rights reserved. Use is subject to &lt;a
-                        href="http://java.net/downloads/jms-spec/Licences/JSR 368 Java Message Service EDR 10.8.15.pdf"&gt;licence
-                        terms&lt;/a&gt; &lt;br/&gt;</bottom>
```

Comment:
1. Do not remove the Oracle copyright notice.
2. The link to the license file should be a link to a "speclicense.html" file that's included in the javadocs.